File: HTO/HTO.Web/Server/Models/SignatureMessage.cs
using HTO.Web.Server.Enums;

namespace HTO.Web.Server.Models
{
	public class SignatureMessage
	{
		public AppTypes AppType { get; set; }
		public decimal Latitude { get; set; }
        public decimal Longitude { get; set; }
        public string LocationImageUrl { get; set; }
		public string Message { get; set; }
        public string SignatureImageDataUrl { get; set; }
		public string Token { get; set; }
	}
}

File: HTO/HTO.Web/Client/Core/Models/signatureMessage.ts
module hto.models {

    "use strict";

    /**
     * Represents a message sent from the mobile application.
     */
    export class SignatureMessage {

        appType: hto.enums.AppTypes;
        latitude: number;
        longitude: number;
        locationImageUrl: string;
        message: string;
        signatureImageDataUrl: string;
        token: string;

    }
}
